pallets/ceres-governance-platform/src/lib.rs

```diff
@@ -12,38 +13,56 @@ mod mock;
 mod tests;
 
 use codec::{Decode, Encode};
-use common::Balance;
+use common::{Balance, BoundedString};
+use frame_support::BoundedVec;
 pub use weights::WeightInfo;
 
 #[derive(Encode, Decode, Default, PartialEq, Eq, scale_info::TypeInfo)]
 #[cfg_attr(feature = "std", derive(Debug))]
-pub struct VotingInfo {
+#[scale_info(skip_type_params(StringLimit))]
+pub struct VotingInfo<StringLimit: sp_core::Get<u32>> {
     /// Voting option
-    voting_option: u32,
+    voting_option: BoundedString<StringLimit>,
```

Comment:
Why not use the option id? You don't really need to save the full option name for each vote

Reply:
Done with Index as voting_option.